packages/suite-desktop/src/support/DesktopUpdater/Available.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { FormattedDate } from 'react-intl';
+import ReactMarkdown from 'react-markdown';
+
+import { Button, Modal, H2, variables, Link } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { Row, LeftCol, RightCol, Divider } from './styles';
+
+import { getReleaseNotes, getReleaseUrl } from '@suite/services/github';
+import { UpdateInfo } from '@suite-types/desktop';
+
+const GreenH2 = styled(H2)`
+    text-align: left;
+    color: ${props => props.theme.TYPE_GREEN};
+`;
+
+const ChangelogWrapper = styled.div`
+    margin: 20px 0px;
+    background: ${props => props.theme.BG_GREY};
+    border-radius: 8px;
+    max-height: 400px;
+    overflow-y: auto;
+    padding: 16px 20px;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    text-align: left;
+
+    h3 {
+        margin-bottom: 6px;
+        font-size: ${variables.FONT_SIZE.NORMAL};
+        font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    }
+
+    h3 {
```

Comment:
this rule seems to be there extra

Reply:
indeed my mistake, @keraf only moved existing styles to a new component.
putting `margin-bottom: 4px;` on line 31 should resolve this if we wanna do it in this PR, but I would let it pass 😬

Reply:
Fixed in 434136c.